### PR TITLE
✨Delete account

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

This PR implements the frontend part of the Account deletion https://github.com/ITISFoundation/osparc-simcore/pull/4871

DeleteAccount

## Related issue/s

- Follows up from https://github.com/ITISFoundation/osparc-simcore/pull/4871
- Implements front end of https://github.com/ITISFoundation/osparc-simcore/issues/4619
- closes https://github.com/ITISFoundation/osparc-simcore/issues/4619

## DevOps

### CODE REVIEW

**Review by pcrespov — APPROVED** [review]
:tada:  bravo! I took the liberty to add a close command in your description for the parent issue

**Review by elisabettai — APPROVED** [review]
Very nice!

So we give 14 days to users to change their mind (in reality because we don't have the entrypoint to delete remove their account from the DB, data, etc...)
If they want deletion right away they send us an e-mail and we delete everything "manually"?

File: services/static-webserver/client/source/class/osparc/desktop/credits/DeleteAccount.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    __form: null,
+
+    _createChildControlImpl: function(id) {
+      let control = null;
+      switch (id) {
+        case "intro-text": {
+          const text = this.tr("\
+            This account will be deleted in 14 days.<br>\
+            If you want to recover during this period, send us an email.\
+            And if you want to delete the data right away, send us an email.\
```
Comment (GitHK) [inline services/static-webserver/client/source/class/osparc/desktop/credits/DeleteAccount.js:44]:
```suggestion
            During this period, if you want to recover it or to delte the\
            data right away, please send us an email.\
```

**Review by GitHK — APPROVED** [review]
👍

**Review by ignapas — APPROVED** [review]
👍🏼 with Andrei's suggestion

**Pull request merged**